### ISSUE TITLE
test is failing

### ISSUE THREAD

**Issue by amitu** [NoRedInk/json-elm-schema#13]
I am getting the following error when running `npm test`:

```sh
➜  json-elm-schema git:(master) npm test                                                                                                                      acko
zsh: correct 'test' to 'tests' [nyae]? n

> elm-json-schema@0.1.0 test /Users/amitu/acko/claimator/json-elm-schema
> tests/run-tests.sh

Success! Compiled 1 module.                                         
Successfully generated /dev/null
/Users/amitu/acko/claimator/json-elm-schema/tests/Main.elm has an invalid module declaration. Check the first line of the file and make sure it has a valid module declaration there!
npm ERR! Test failed.  See above for more details.
```

npm: 4.9.0, node: v7.9.0, yarn: v0.16.1.

I am trying to create a pull request to add [maxProperties](http://json-schema.org/latest/json-schema-validation.html#rfc.section.6.15) and `minProperties` to `object`.

**Comment by jwoudenberg** [comment]
Looks like it complains about `tests/Main.elm` being invalid. What happens if you run `elm-make` on it?

**Comment by amitu** [comment]
```
➜  tests git:(master) elm-make Main.elm
Success! Compiled 1 module.
Successfully generated index.html
➜  tests git:(master) elm-make -v 
elm-make 0.18 (Elm Platform 0.18.0)
```

I am running it on master branch.

**Comment by amitu** [comment]
Tried it on a colleague's machine too, there too its not working. Exact steps:

```sh
$ git clone git@github.com:NoRedInk/json-elm-schema.git
$ cd json-elm-schema
$ npm install
$ elm-package install
$ cd tests
$ elm-package install
$ cd ..
$ npm test
```

Both machines I tried are OSX 10.12.5.

**Comment by jwoudenberg** [comment]
I managed to reproduce this by clearing my own dependencies. Would you mind retrying `npm test` I created a PR that fixes it for me: #16. You might need to rerun `npm install` first, and clear your `elm-stuff` in the main project and in the `tests` directory.

Would you mind giving it a try on your machine to see if it works for you too? If so I'll merge.

And thanks a lot for reporting!

**Comment by amitu** [comment]
Thank you for releasing such an awesome thing, and all the prompt replies, even on weekend. Really appreciate it.
